Makefile

```diff
@@ -103,6 +103,8 @@ FISHINSTALLDIR=${PREFIX}/share/fish/vendor_completions.d
 
 SELINUXOPT ?= $(shell test -x /usr/sbin/selinuxenabled && selinuxenabled && echo -Z)
 
+MACHINE_POLICY_JSON_DIR ?= .
```

Comment:
I don't think we should set the default for everything  to `.`, that just doesn't make sense for local development, i.e. see https://github.com/containers/podman/issues/21855

Using `$(pwd)/pkg/machine/ocipull` and then overwriting this to `.` for the zip file target should be better IMO.

Reply:
Anyhow it doesn't make the current situation any worse though so consider this non blocking.

Reply:
IMO a binary you build with podman-remote should work just as well (and ideally the same) as the one installed by a package (and behave similarly). It's a pain to have to build an installer just to try out a change, or to have always pass custom flags to podman-remote (that are always the same).  For example, a common workflow a lot of folks use is to just copy up a new binary in the same location on the path. Another example is we had CI tasks that used the binary built from podman-remote (and not the full zip), and so the different behavior was an issue there and required installing a local override (not a huge deal, but it just illustrates the issue). IMO the issue is just another form of it

The path you link above is also not quite right because it only works on a system that was building the binary, you can't copy it to another system or hand to someone else to test and have it work. 

I think the only thing we can do to address this reliably is to search multiple locations, which is at least already the case on Linux (where /etc/containers is likely to exist), or that other idea you had to possibly fetch a policy from somewhere if it doesnt exist.

Reply:
Except a binary compiled on a dev machine running with `bin/podman` won't work because the policy.json is not in the same directory as in the binary.